perses/app/setup_relative_calculation.py

```diff
@@ -517,26 +519,22 @@ def run_setup(setup_options):
                 endstates = True
             #TODO expose more of these options in input
             if setup_options['fe_type'] == 'sams':
-                hss[phase] = HybridSAMSSampler(mcmc_moves=mcmc.LangevinSplittingDynamicsMove(timestep=timestep,
-                                                                                             collision_rate=5.0 / unit.picosecond,
-                                                                                             n_steps=n_steps_per_move_application,
-                                                                                             reassign_velocities=False,
-                                                                                             n_restart_attempts=20,
-                                                                                             splitting="V R O R V",
-                                                                                             constraint_tolerance=1e-06),
+                hss[phase] = HybridSAMSSampler(mcmc_moves=mcmc.LangevinDynamicsMove(timestep=timestep,
+                                                                                    collision_rate=5.0 / unit.picosecond,
```

Comment:
I think we want to use 1/ps for the collision rate unless we have a compelling reason to use a higher rate.